projectroles/tests/test_permissions_ajax.py

```diff
@@ -237,6 +236,101 @@ def test_get_category_anon(self):
         self.assert_response(self.url_cat, self.anonymous, 401, method='POST')
 
 
+class TestSidebarContentAjaxView(ProjectPermissionTestBase):
+    """Tests for SidebarContentAjaxView permissions"""
+
+    def setUp(self):
+        super().setUp()
+        self.url = reverse(
+            'projectroles:ajax_sidebar',
+            kwargs={'project': self.project.sodar_uuid},
+        )
+        self.url_cat = reverse(
+            'projectroles:ajax_sidebar',
+            kwargs={'project': self.category.sodar_uuid},
+        )
+
+    def test_get(self):
+        """Test SidebarContentAjaxView GET"""
+        good_users = [
+            self.superuser,
+            self.user_owner_cat,
+            self.user_delegate_cat,
+            self.user_contributor_cat,
+            self.user_guest_cat,
+            self.user_owner,
+            self.user_delegate,
+            self.user_contributor,
+            self.user_guest,
+        ]
+        bad_users = [self.user_no_roles, self.user_finder_cat, self.anonymous]
```

Comment:
Let's follow the conventions for ordering the users for better readability: first all the users with roles, then user_no_roles, ending with anonymous.